reference/safe-core-sdk/auth-kit/web3auth-modal.md

```diff
@@ -0,0 +1,89 @@
+## Web3AuthModal
+
+Web3Auth is a pluggable authentication infrastructure for Web3 wallets and applications. It simplifies onboarding for both mainstream and crypto native users, offering experiences tailored to their preferences. It supports all social logins, web and mobile native platforms, wallets, and other key management methods, creating a cryptographic key provider specific to the user and application.
+
+The `Web3AuthModalPack` enables you to use [Web3Auth modal](https://web3auth.io/docs/sdk/web/modal/) SDK with added Safe capabilities, such as retrieving the related Safe addresses for an external owned account created using Web3Auth services.
+
+### Install
+
+To use the `Web3AuthModalPack`, you need to install additional dependencies in addition to the `@safe-global/onramp-kit` package.
+
+```bash
+yarn add @web3auth/base @web3auth/modal
```

Comment:
```suggestion
yarn add @web3auth/base @web3auth/modal @safe-global/onramp-kit
```

If someone gets directly into this section they won't forget to install the onramp-kit. If they already added it it won't harm.

Reply:
ok